scripts/cybershake/install_cybershake_fault.py

```diff
@@ -159,7 +159,12 @@ def install_fault(
     vel_mod_dir = simulation_structure.get_fault_VM_dir(root_folder, fault_name)
     vm_params_path = os.path.join(vel_mod_dir, VM_PARAMS_FILE_NAME)
     if check_vm:
-        valid_vm, message = validate_vm.validate_vm(vel_mod_dir, srf=list_srf[0])
+        valid_vm, message = validate_vm.validate_vm_params(vm_params_path)
```

Comment:
Ahhh, they are different. Could we use different variable names for clarity of reading 

valid_vm_params, valid_vm_files etc?